issue-rules.yml

```diff
@@ -55,24 +55,33 @@ rules:
   addLabels: ['Area: Core']
 - valueFor: '**Enter Task Name**'
   contains: 'PowerShellV2'
-  addLabels: ['Area: ABTT', 'triage']
+  addLabels: ['Area: ABTT', 'Task: PowerShell', 'triage']
 - valueFor: '**Enter Task Name**'
   contains: 'SSH'
   addLabels: ['Area: ABTT', 'triage']
+- valueFor: '**Enter Task Name**'
+  contains: 'CopyFilesOverSSH'
+  addLabels: ['Area: ABTT', 'Task: CopyFilesOverSSH', 'triage']
+- valueFor: '**Enter Task Name**'
+  contains: 'InstallSSH'
+  addLabels: ['Area: ABTT', 'Task: InstallSSHKey', 'triage']
 # Area: CrossPlatform
 - valueFor: '**Enter Task Name**'
   contains: 'Android'
   addLabels: ['Area: ABTT', 'triage']
+- valueFor: '**Enter Task Name**'
+  contains: 'AndroidSigning'
+  addLabels: ['Task: AndroidSigning']
```

Comment:
Should the ABTT label get assigned here too?

Reply:
Yep thanks 👍  added 'Area: ABTT' label also here